baseframe/forms/fields.py

```diff
@@ -391,7 +391,6 @@ def iter_choices(self):
             return [(u.userid, u.pickername, True) for u in self.data]
 
     def process_formdata(self, valuelist):
-        retval = super(UserSelectFieldBase, self).process_formdata(valuelist)
```

Comment:
This base class is a subclass of `object`. This is unnecessary. `process_formdata()` doesn't seem to need to return anything.

Reply:
If what you say is true, this line should have failed because `object` does not provide a `process_formdata`. The line works because this is a mixin class and some other class in the hierarchy actually does provide this method.

Please restore this line.